### DISCUSSION TITLE
Error Receivers -> filelog -> operators: type: file_input

### DISCUSSION THREAD

**Discussion by WagnerCarvalho** [open-telemetry/opentelemetry-collector-contrib#12182] [Q&A]
hey guys, I'm getting error **operator 'teste' can not process entries: {"operator_id":"file_input"}** when uploading the sidcard in my application. Please could someone help me?

![Screen Shot 2022-07-08 at 12 03 23](https://user-images.githubusercontent.com/7527649/178019064-93181f63-623b-4428-bb7d-eadde077b1fd.png)

![Screen Shot 2022-07-08 at 12 13 15](https://user-images.githubusercontent.com/7527649/178020737-b8ee2b6f-ec90-4ef6-b61a-d11c0724f1b6.png)

**Comment by jaydeepp** [comment]
Hi, Were you able to solve the issue?
